src/applications/gi/sass/partials/_gi-landing-page.scss

```diff
@@ -29,5 +29,6 @@
 
   .type-of-institution-filter .form-expanding-group-open {
```

Comment:
only place needing changes to this class are the "Type of institution" filter on the Landing Page